modules/gapi/test/gapi_statefulkernel_test_utils.hpp

```diff
@@ -0,0 +1,46 @@
+// This file is part of OpenCV project.
```

Comment:
RE file name. There is no such thing as `gapi_statefulkernel`.

It is not a G-API kernel but an OpenCV Backend Kernel.

Reply:
Renamed to gapi_ocv_kernel_st_test_utils.hpp (name is derived from macro name for OCV Stateful kernel).

Reply:
HERE better to put "stateful" instead of just "st". Looking at the directory listing, go figure out what this file is about?

Reply:
Fixed.